app/org/sagebionetworks/bridge/services/UploadService.java

```diff
@@ -236,12 +259,10 @@ public Upload getUpload(@Nonnull String uploadId) {
             throw new BadRequestException("Query window cannot be longer than two days: " + startTime + "-" + endTime);
         }
```

Comment:
Changed. Changed tests as well to test participant/study uploads calls separately.